cmd/geninstaller/install.sh.tmpl

```diff
@@ -86,6 +86,11 @@ To create a new Hermit environment run:
 
   $HERMIT_BIN_INSTALL_DIR/hermit init <dir>
 
+To setup shell hooks to automatically activate your environment when you enter
+it, run:
+
+  $HERMIT_BIN_INSTALL_DIR/hermit shell-hooks --<zsh | bash>
```

Comment:
Do shell hooks work if `hermit` isn't in the $PATH?

Reply:
Works on my machine!